GOVERNANCE.md

```diff
@@ -0,0 +1,216 @@
+- [Values](#values)
+- [Community Processes](#community-processes)
+- [Roles](#roles)
+  + [Contributors](#Contributors)
+  + [Maintainers](#Maintainers)
+  + [Release Managers](#Release-Managers)
+  + [Admins](#Admins)
+  + [GitHub Owners](#GitHub-Owners)
+  + [Moderators](#Moderators)
+- [Decision Making](#decision-making)
+- [Meetings](#meetings)
+- [Code of Conduct](#code-of-conduct)
+- [Security Response Team](#security-response-team)
+- [Modifying this Charter](#modifying-this-charter)
+<!-- TOC end -->
+
+## Values
+
+The RIOT community embraces the following values:
+
+* **Openness:** Communication and decision-making happens in the open and is discoverable for future reference.
+  As much as possible, all discussions and work events eventually take place in public forums and open repositories.
+
+* **Fairness:** All stakeholders have the opportunity to provide feedback and submit contributions, which will be considered on their merits.
+
+* **Community over Product or Company:** Sustaining and growing our community takes
+  priority over shipping code or sponsors' organizational goals.
+  Each contributor participates in the project as an individual.
+
+* **Inclusivity:** We innovate through different perspectives and skill sets, which can only be accomplished in a welcoming and respectful environment.
+
+* **Participation:** Responsibilities within the project are earned through participation, and there is a clear path up the contributor ladder into more responsible positions.
+
+
+## Community Processes
+The community around RIOT gathers many IoT developers and users from around the world, from the industry, from academia, and hobbyists.
+The RIOT community is open to everyone.
+Our channels to communicate can be found in our [contributing guidelines].
+The community self-organizes using the roles described below.
+
+
+## Roles
+### Contributors
+Contributors are people who contribute there work to RIOT.
+This includes,
+
+- of course, code contributions, but also
+- bug reports,
+- other issue reports,
+- participation in technical as well as non-technical discussions, or
+- organizational considerations.
+
+Code contributions are very welcome, but in order to streamline and harmonize code quality, contributors must follow the [contributing guidelines].
+
+
+### Maintainers
+
+Among contributors, some have maintainer status, which consists in rights (write access to the [RIOT GitHub repository](https://github.com/RIOT-OS/RIOT/)) and duties.
+The current maintainers can be found in the [maintainers list].
+
+The duties of a maintainer include the following: Maintainers are people who care about RIOT and want to help it grow and improve.
+A maintainer is not just someone who can make changes, but someone who has demonstrated their ability to collaborate with the team, get the most knowledgeable people to review code and docs, contribute high-quality code, and follow through to fix issues (in code or tests).
```

Comment:
Did we have this wording before? It basically makes "contribute high-quality code" a prerequisite and would exclude e.g., @jkarinkl or @waehlisch from being maintainers. Which is probably not the intention?

Reply:
No that seem to have come from the [GOVERNANCE-maintainer.md template](https://github.com/cncf/project-template/blob/main/GOVERNANCE-maintainer.md#maintainers).

Reply:
Would it be more helpful to make the "and" an "or"? I actually like having some specifics on “what makes a good maintainer” in here.

Reply:
I'd be happy to add any proposal :-)

Reply:
```suggestion
A maintainer is not just someone who can make changes, but someone who has demonstrated their ability to collaborate and organize with the team, get the most knowledgeable people to review code or documentation, contribute high-quality code or documentation, as well as follow through to fix issues (in code, tests, or tests).
```

would be my edit to that.